src/services/asyncStorage.ts

```diff
@@ -0,0 +1,56 @@
+//IMPORTS//
+import AsyncStorage from "@react-native-async-storage/async-storage";
+
+/** Sets Data in Local Storage */
+export const setDataInLocalStorage = async (
+  key: string,
+  value: string | Array<any> | object
+): Promise<boolean> => {
+  try {
+    const serializedValue = JSON.stringify(value);
+    await AsyncStorage.setItem(key, serializedValue);
+    return true;
+  } catch (error) {
+    console.error(error);
+    return false;
+  }
+};
+
+/** Gets Data from Local Storage */
+export const getDataFromLocalStorage = async (
+  key: string
+): Promise<any | undefined> => {
+  try {
+    const value = await AsyncStorage.getItem(key);
+    return value ? JSON.parse(value) : undefined;
+  } catch (error) {
+    console.error(error);
+    throw error;
+  }
+};
+
+/** Remove Data from Local Storage */
+export const removeDataFromLocalStorage = async (key: string): Promise<void> => {
+  try {
+    await AsyncStorage.removeItem(key);
+  } catch (error) {
+    console.error(error);
+    throw error;
+  }
+};
+
+/** Remove Multiple Data/Keys from Local Storage */
+export const removeMultipleDataFromLocalStorage = async (): Promise<void> => {
```

Comment:
change function to flushLocalStorage

And remove all the keys from Local Storage, do not exclude anything